tools/terraform/plugin.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.1
+tools:
+  downloads:
+    - name: terraform
```

Comment:
Is this going to collide with the terraform linter download? A bit worried about cli internals here

Reply:
fair enough - we won't have a collision, but this should be resolved somehow anyway